react/components/PopoverContext.tsx

```diff
@@ -48,6 +59,11 @@ function popoverContextReducer(state: State = DEFAULT_STATE, action: Action) {
         ...state,
         containerRef: action.payload?.containerRef,
       }
+    case 'SET_TRIGGER_MODE':
+      return {
+        ...state,
+        triggerMode: action.payload?.triggerMode,
```

Comment:
This could cause you to set `triggerMode` to `undefined`, right? That might not be what you want.